packages/inventory/src/components/table/InventoryTable.js

```diff
@@ -30,9 +31,13 @@ const InventoryTable = forwardRef(({
     getEntities,
     hideFilters,
     paginationProps,
+    errorState = <ErrorState />,
     ...props
 }, ref) => {
     const hasItems = Boolean(items);
+    const error = useSelector(({ entities: { error } }) => (
+        error
+    ));
```

Comment:
I would use safeAccessor here. It could happen that entities are not an object if app missregistered the reducder
```suggestion
    const error = useSelector(({ entities }) => (
        entities?.error
    ));
```